static/src/stylesheets/module/commercial/_adverts.scss

```diff
@@ -737,10 +737,10 @@
         float: none;
         min-height: 0;
         overflow: visible;
-        margin: auto $gs-gutter / 4 0;
```

Comment:
I changed it a little now. I want the logo to be in the right bottom corner. The width is 100% and I added extra padding-right in the div inside.